wren-ui/src/apollo/server/repositories/threadResponseRepository.ts

```diff
@@ -105,11 +113,16 @@ export class ThreadResponseRepository
       throw new Error('Unexpected dbdata');
     }
     const camelCaseData = mapKeys(data, (_value, key) => camelCase(key));
-    // JSON.parse detail and error
-    return {
-      ...camelCaseData,
-      detail: camelCaseData.detail ? JSON.parse(camelCaseData.detail) : null,
-      error: camelCaseData.error ? JSON.parse(camelCaseData.error) : null,
-    } as ThreadResponse;
+    const formattedData = mapValues(camelCaseData, (value, key) => {
+      if (['error', 'detail'].includes(key)) {
+        if (typeof value === 'string') {
```

Comment:
The value from Sqlite will be string type, while the value from PG is JSON object

Reply:
Maybe add a comment to this line?